src/app/core/admin/user-management/admin-list-users.component.html

```diff
@@ -149,7 +149,10 @@ <h1 skipTo>
 									>
 								</li>
 								<li>
-									<a class="dropdown-item" (click)="confirmDeleteUser(user)"
+									<a
+										*ngIf="user.userModel.name != this.session.name"
```

Comment:
@smasucci - although it may seem odd, we should let logged in users delete their own account. In certain environments, these accounts will be auto-created from other authentication mechanisms (e.g., OAuth or PKI credentials), and we wouldn't want to prevent an admin from taking this action.

Reply:
alright! in that case we don't need to get the session (it was only to check the name of the current user) so i'll take out all of those changes